frontend/src/components/AlterSchemaPrepForm/AlterSchemaPrepForm.vue

```diff
@@ -100,7 +102,7 @@ import {
   Repository,
   UNKNOWN_ID,
 } from "../../types";
-import { sortDatabaseList } from "../../utils";
+import { allowGhostMigration, isDev, sortDatabaseList } from "../../utils";
```

Comment:
* The imports order could be re-sort;
* We can use `@` instead of `"../../"`;

Reply:
Updated, PTAL